src/routes/Board/BoardGuard.tsx

```diff
@@ -37,7 +37,7 @@ export const BoardGuard = ({printViewEnabled}: BoardGuardProps) => {
     return <Board />;
   }
 
-  if (boardStatus === "passphrase_required") {
+  if (boardStatus === "passphrase_required" || boardStatus === "too_many_join_requests") {
```

Comment:
Spamming the f5 leads to the passphrase dialog in a manual approval session. No matter whether the user was accepted or rejected before. Not sure if this is intended behavior.


https://user-images.githubusercontent.com/44020029/201974425-e3f76b8d-1a65-4689-96c4-fc4853af2159.mov

Additionally, the greyed out button doesn't disable the submitting (pressing Enter works fine)